docker/superset/README.md

```diff
@@ -0,0 +1,136 @@
+=======
+
+# <a name="table-of-contents"></a>Table of Contents
+* [Using Docker](#using-docker)
+  * [Running Superset](#running-Superset-docker)
+    * [Running Superset with SQLite](#Runnung-Superset-SQLite)
+    * [Running Superset with additional parameters](#Running-Superset-with-additionlal-parameters)
+* [References](#references)
+  * [Ports](#references-ports)
+  * [Environment Variables](#references-environment-variables)
+  * [Volumes](#references-volumes)
+
+# <a name="using-docker"></a>Using Docker
+
+Consult [Marketplace container documentation](https://cloud.google.com/marketplace/docs/container-images)
+for additional information about setting up your Docker environment.
+
+## <a name="running-Superset-docker"></a>Running Superset
+
+This section describes how to spin up an Superset service using this image.
+
+### <a name="Runnung-Superset-SQLite"></a>Running Superset with SQLite
+
+Use the following content for the `docker-compose.yml` file, then run `docker-compose up`.
+
+```yaml
+version: '2'
+services:
+  hazelcast:
+    image: marketplace.gcr.io/google/superset1
+    ports:
+      - 8088:8088
+```
+
+Or you can use `docker run` directly:
+
+```shell
+docker run -d \
+    -p 8088:8088 \
+    --name superset \
+    marketplace.gcr.io/google/superset1
+```
+
+### <a name="Running-Superset-with-additionlal-parametersr"></a>Running Superset with additional parameters
+
+Use the following content for the `docker-compose.yml` file, then run `docker-compose up`.
+
+
+```yaml
+version: '2'
+services:
+  hazelcast:
+    image: marketplace.gcr.io/google/superset1
+    ports:
+      - 8088:8088
+    environment:
+      SUPERSET_PASSWORD: superset/some-password
+      SUPERSET_LOAD_EXAMPLES: yes            
+```
+
+Or you can use `docker run` directly:
+
+```shell
+docker run -d \
+    -p 8088:8088 \
+    --name hazelcast \
+    -e SUPERSET_PASSWORD="superset/some-password" \
+    -e NEO4J_dbms_logs_debug_level=yes \
+    marketplace.gcr.io/google/superset1
+```
+
+# <a name="references"></a>References
+
+## <a name="references-ports"></a>Ports
+
+These are the ports exposed by the container image.
+
+| **Port** | **Description**    |
+| :------- | :----------------- |
+| TCP 8088 | Superset Dashboards|
+
+## <a name="references-environment-variables"></a>Environment Variables
+
+These are the environment variables understood by the container image.
+| **Variable**                  | **Description**                                                                                        |
+| :---------------------------- | :----------------------------------------------------------------------------------------------------- |
+| SUPERSET_PASSWORD     | Sets password for Admin user while initializing Superset  |
+| SUPERSET_LOAD_EXAMPLES | Dowloands and install some Doshaboard examples to play with.              |
```

Comment:
download. dashboards. 

remove `some` and to `play with`

Reply:
done